String locationName
___

Position coordinates
bool success
